.devcontainer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/devcontainers/miniconda:0-3
```

Comment:
```suggestion
FROM continuumio/miniconda3:latest
```

Reply:
Well, it works, but it leaves you with a container that is run as root, whereas the MSFT version is run as the user `vscode`---arguably a better approximation of what a user might find. It achieves that not by directly deriving from our image, but by copying only the `/opt/conda` folder from `continuumio/miniconda3:latest` into the new image, which is based on `mcr.microsoft.com/devcontainers/base:1-bullseye`.